src/styles/semantic-tokens.css

```diff
@@ -132,7 +123,11 @@
 
     --background: var(--black);
     --background-highlight: var(--gray-900);
-    --background-medium: var(--gray-950); /* TODO: VERIFY */
+
+    /* TODO: Add "low" "medium" and "high" to design system */
```

Comment:
nitpick: would avoid adding todos that are outside of the scope of the codebase as they are hard to maintain and easy to forget.

Reply:
Yes yes, sounds like we have consensus and approval on this proposal, so I'll go ahead and remove that and add notes in the DS